config/opt.py

```diff
@@ -0,0 +1,63 @@
+from typing import Optional, Literal
+from zencfg import ConfigBase
+
+class OptimizationConfig(ConfigBase):
```

Comment:
I thought we decided to not have docstrings for these. Also they're not parameters technically.